src/prepack-cli.js

```diff
@@ -227,16 +227,35 @@ function run(
     let serialized = prepackFileSync(inputFilenames, resolvedOptions);
     processSerializedCode(null, serialized);
   } catch (x) {
-    console.error(x.message);
-    console.error(x.stack);
-    process.exit(1);
+    //FatalErrors must have generated clear CompilerDiagnostic and can safely go to the next codeblock.
+    if (x instanceof FatalError) {
+      invariant(errors.size > 0, "FatalError must generate at least one CompilerDiagnostic");
+    } else {
+      console.error(x.stack);
+      process.exit(1);
```

Comment:
It would be better if any diagnostics that were collected were printed out before printing the stack trace and terminating the process. One way to do this it make the catch wrap the finally.

Reply:
Yes.